atom/browser/native_window.cc

```diff
@@ -317,6 +317,17 @@ void NativeWindow::InspectElement(int x, int y) {
   agent->InspectElement(x, y);
 }
 
+void NativeWindow::InspectServiceWorker() {
```

Comment:
All the devtools api we have listed are on `BrowserWindow` besides if they use `window.webcontents` before its finished loading wouldnt `webcontents.opendevtools` crash ?